bugshaker/src/main/java/com/github/stkent/bugshaker/MainActivity.java

```diff
@@ -0,0 +1,279 @@
+import com.github.stkent.bugshaker.flow.widget.DrawingView;
+import com.github.stkent.bugshaker.utilities.LogcatUtil;
+import com.github.stkent.bugshaker.utilities.Logger;
+import com.github.stkent.bugshaker.utilities.Toaster;
+
+public class MainActivity extends AppCompatActivity implements OnClickListener {
+
+	private DrawingView drawView;
+
+	private FeedbackEmailFlowManager feedbackEmailFlowManager;
+
+
+
+	private Activity activity;
+
+	private float smallBrush, mediumBrush, largeBrush, smallEraser, mediumEraser, largeEraser;
+	private ImageButton currPaint;
+
+
+	@Override
+	protected void onCreate(Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		setContentView(R.layout.activity_main);
+		activity = this;
+
+		drawView = (DrawingView)findViewById(R.id.drawing);
+		LinearLayout paintLayout = (LinearLayout)findViewById(R.id.paint_colors);
+		currPaint = (ImageButton)paintLayout.getChildAt(0);
+		currPaint.setImageDrawable(getResources().getDrawable(R.drawable.paint_pressed));
+
+		smallBrush = getResources().getInteger(R.integer.small_size);
+		mediumBrush = getResources().getInteger(R.integer.medium_size);
+		largeBrush = getResources().getInteger(R.integer.large_size);
+
+		smallEraser = getResources().getInteger(R.integer.small_eraser_size);
+		mediumEraser = getResources().getInteger(R.integer.medium_eraser_size);
+		largeEraser = getResources().getInteger(R.integer.large_eraser_size);
+
+		ImageButton drawBtn = (ImageButton) findViewById(R.id.draw_btn);
+		drawBtn.setOnClickListener(this);
+
+		drawView.setBrushSize(smallBrush);
+
+		ImageButton eraseBtn = (ImageButton) findViewById(R.id.erase_btn);
+		eraseBtn.setOnClickListener(this);
+
+		ImageButton sendBtn = (ImageButton) findViewById(R.id.sendEmail);
+		sendBtn.setOnClickListener(this);
+
+		ImageButton speechBtn = (ImageButton) findViewById(R.id.speechBox);
+		speechBtn.setOnClickListener(this);
+
+		final GenericEmailIntentProvider genericEmailIntentProvider
+			= new GenericEmailIntentProvider();
+
+		Logger logger = new Logger(true);
+		Application application = getApplication();
+
+		EmailCapabilitiesProvider emailCapabilitiesProvider = new EmailCapabilitiesProvider(
+			application.getPackageManager(), genericEmailIntentProvider, logger);
+
+		feedbackEmailFlowManager = new FeedbackEmailFlowManager(
+			application,
+			emailCapabilitiesProvider,
+			new Toaster(application),
+			new ActivityReferenceManager(),
+			new FeedbackEmailIntentProvider(application, genericEmailIntentProvider));
+
+		String pathOfScreenshot = getIntent().getStringExtra("uri");
+		File screenshotFile = new File(pathOfScreenshot);
+		if(screenshotFile.exists()) {
+			Bitmap myBitmap = BitmapFactory.decodeFile(pathOfScreenshot);
+
+			drawView = (DrawingView) findViewById(R.id.drawing);
+			Drawable temp = new BitmapDrawable(getResources(), myBitmap);
+			drawView.setBackgroundDrawable(temp);
+		}
+		else {
+			throw new RuntimeException();
```

Comment:
We don't need this anymore. Maybe we should log a comment that the file doesn't exist!!